src/NewTwitchApi/Resources/AbstractResource.php

```diff
@@ -21,27 +21,103 @@ public function __construct(Client $guzzleClient)
     /**
      * @throws GuzzleException
      */
-    protected function callApi(string $uriEndpoint, string $bearer, array $queryParamsMap = []): ResponseInterface
+    protected function callApi(string $uriEndpoint, string $bearer, array $queryParamsMap = [], array $bodyParams = []): ResponseInterface
```

Comment:
With `put`, `post`, and `delete` being the other methods, we should really name this one `get`. Not now, of course. That will be a breaking change. We can do it for the next major release. Please ticket? :-)

Reply:
#92 92